src/fs/rock/RockRebuild.cc

```diff
@@ -695,18 +698,27 @@ Rock::Rebuild::swanSong()
 }
 
 void
-Rock::Rebuild::failure(const char *msg, int errNo)
+Rock::Rebuild::callException(const std::exception &)
 {
+	throw;
```

Comment:
It looks like this comment was missed in an earlier suggestion (probably because that suggestion had a "thow" typo):

```suggestion
    // For now, treat all Rebuild exceptions as fatal errors rather than letting
    // the default callException() implementation to silently stop this job.
	throw;
```

Reply:
Sorry, I forgot to add this comment while figuring out which of the two suggestions to this method was relevant.